mne/dipole.py

```diff
@@ -445,11 +470,22 @@ def fit_dipole(evoked, cov, bem, trans=None, n_jobs=1, verbose=None):
     """
     # This could eventually be adapted to work with other inputs, these
     # are what is needed:
+
+    evoked.info['projs'] = []  # get rid of SSP projections
+    if 'eeg' in evoked:
+        evoked = evoked.copy()
+        avg_proj = make_eeg_average_ref_proj(evoked.info)
+        evoked.add_proj([avg_proj])
+        evoked.apply_proj()
```

Comment:
why get rid of existing projs?

I think we should replicate the proj check that is used in minimum norm inverses, which includes a warning about average EEG ref (not forcing it to be there) IIRC